ubuntu-headless-qt/script/create_rzpi_user.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Check if the script is run as root
+if [ "$(id -u)" -ne 0 ]; then
+    echo "Please run this script as root or using sudo."
+    exit 1
+fi
+
+# Variables
```

Comment:
Should we move these variables into a "configuration" file? So Users can set them to whatever they want instead of hardcoding.

Reply:
User can create themselves a new user so I will remove this feature